Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -327,5 +338,21 @@ public CacheContext(CacheProcessorTests owner, string commandLineArgs = "/k:key"
             public void Dispose() =>
                 Sut.Dispose();
         }
+
+        private sealed class FileData
```

Comment:
Using records could have made this a lot simpler but the project targets net48. We should reconsider if targeting `net48` is still needed.